.github/workflows/build-multi-arch.yml

```diff
@@ -21,6 +21,9 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Clean up docker
+      run:  docker system prune -a --volumes
+
     - name: Clean up stale docker images
       run: sudo docker image prune -f
```

Comment:
Could you also remove sudo from here, if not needed ?

Reply:
removed.